lib/mongoid/findable.rb

```diff
@@ -116,6 +116,36 @@ def exists?(id_or_conditions = :none)
       with_default_scope.exists?(id_or_conditions)
     end
 
+    # Return true if any documents exist in the criteria.
+    #
+    # @example Determine if any documents exist
+    #   criteria.any?
```

Comment:
I know this is an artifact from when you had this on `Mongoid::Criteria`, but for documentation purposes you can just reference `Model`, generically.

```suggestion
    #   Model.any?
```

Ditto for the other new methods you added.